style.css

```diff
@@ -173,4 +173,57 @@ body {
   text-decoration: underline;
 }
 
+#twelveArticles {
```

Comment:
When choosing names, you should be thinking generally and how to reuse stuff. For example, here you named a div `twelveArticles`. What happens if we decide to show 8 instead of 12 articles? Are we going to change every place where it was used (same for `twelveArticlePic` etc.).

A better approach would be to have to some reusable names, for example
1) `gridMediumArticles`
2) `mediumArticleImg`
3) 'mediumArticleHeader`